### ISSUE TITLE
During fine-tuning, do we freeze the SSL pretrained encoder backbone for the initial few steps?

### ISSUE THREAD

**Issue by kaushal-py** [Open-Speech-EkStep/vakyansh-nemo-experimentation#2]
Hi, thanks for releasing the code and pretrained SSL checkpoints for conformer models.

When performing finetuning from the SSL pretrained checkpoint, I observed that the encoder backbone is not frozen during the initial steps. Is my understanding correct?
This might be an issue as the network may get large gradients during the initial part of fine-tuning, and may lead to catastrophic forgetting of the encoder weights which were learnt during SSL pre-training.  In wav2vec-style models, we used to freeze the encoder weights to tackle this issue.
Did you observe that starting from the SSL pretrained checkpoint led to better performance on benchmarks?
